Tools/machines/adastra-cines/submit.sh

```diff
@@ -31,6 +31,14 @@ export MPICH_GPU_SUPPORT_ENABLED=1
 # or, less invasive:
 export FI_MR_CACHE_MONITOR=memhooks  # alternative cache monitor
 
+# note
+# On machines with similar architectures (Frontier, OLCF) these settings
+# seem to prevent the following issue:
+# OLCFDEV-1597: OFI Poll Failed UNDELIVERABLE Errors
+# https://docs.olcf.ornl.gov/systems/frontier_user_guide.html#olcfdev-1597-ofi-poll-failed-undeliverable-errors
```

Comment:
@lucafedeli88 do you want to add a `Known issues` section for Adastra's README file as we do e.g., here?
https://warpx.readthedocs.io/en/latest/install/hpc/frontier.html#known-system-issues